sql/mob_pools.sql

```diff
@@ -760,7 +760,7 @@ INSERT INTO `mob_pools` VALUES (699,'Chasmic_Hornet','Chasmic_Hornet',48,0x00001
 INSERT INTO `mob_pools` VALUES (700,'Chasm_Coeurl','Chasm_Coeurl',71,0x00006F0100000000000000000000000000000000,1,1,2,240,100,0,1,0,0,0,0,0,0,0,0,0,0,1,0,71);
 INSERT INTO `mob_pools` VALUES (701,'Chasm_Gnat','Chasm_Gnat',131,0x0000DD0700000000000000000000000000000000,6,6,3,240,100,0,1,0,0,0,0,0,0,0,0,0,0,1,0,131);
 INSERT INTO `mob_pools` VALUES (702,'Chasm_Lizard','Chasm_Lizard',174,0x0000480100000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,565,131,0,0,0,1,0,174);
-INSERT INTO `mob_pools` VALUES (703,'Chatoyant_Quadav','Chatoyant_Quadav',202,0x00001E0400000000000000000000000000000000,3,3,6,265,100,0,1,0,1,0,0,0,1,1157,0,0,1,1,0,202);
+INSERT INTO `mob_pools` VALUES (703,'Chatoyant_Quadav_BS','Chatoyant_Quadav_BS',202,0x00001E0400000000000000000000000000000000,3,3,6,265,100,0,1,0,1,0,0,0,1,131,0,0,1,1,0,202);
```

Comment:
"BS"?

Reply:
_BS = Beadeaux [S].  Some of the mobs needed multiple pools even within past zones, like the ones in Beadeaux [S] and the ones in Castle Zvahl Keep [S] have different flags and/or models.

Reply:
got it, thanks.